cli/tools/coverage/mod.rs

```diff
@@ -43,33 +43,32 @@ impl CoverageCollector {
   }
 
   async fn enable_debugger(&mut self) -> Result<(), AnyError> {
-    self.session.post_message("Debugger.enable", None).await?;
+    self.session.post_message("Debugger.enable", ()).await?;
```

Comment:
Why is this change necessary? `None` reflects intent better when there are no parameters than `()`; could we revert back to the previous API?